docs/flink/flink-getting-started.md

```diff
@@ -547,6 +547,64 @@ RewriteDataFilesActionResult result = Actions.forTable(table)
 
 For more doc about options of the rewrite files action, please see [RewriteDataFilesAction](../../../javadoc/{{% icebergVersion %}}/org/apache/iceberg/flink/actions/RewriteDataFilesAction.html)
 
+## Type compatibility
+
+Flink and Iceberg support different set of types. Iceberg does the type conversion automatically, but not for all combinations,
```

Comment:
Question: for what combinations can Iceberg not do the conversion automatically? From the perspective of a new user, this might leave them with more questions than answers.